### PR TITLE
Add talk reminder bot

### LINKED ISSUE #69: Current talk discord bot
It would be great to have a service that posts the talk name and speaker at the beginning of the talk to the relevant discord channel.

### PR DESCRIPTION
**Pull request by christopher-dG**
Closes #69 

Pretty basic proof of concept so far, still some polishing to be done.

### CODE REVIEW

**Comment by christopher-dG** [comment]
Capture

**Comment by travigd** [comment]
@aviks Can we deploy this to a micro AWS instance?

**Comment by aviks** [comment]
> @aviks Can we deploy this to a micro AWS instance?

Yeah, sure. I'll set one up tomorrow morning.

**Comment by christopher-dG** [comment]
Capture


Thoughts?

**Comment by travigd** [comment]
@christopher-dG Are you looking for a review on this? I'm not sure how WIP-y it is.

**Comment by christopher-dG** [comment]
It's basically ready to go now.

**Comment by christopher-dG** [comment]
Tomorrow I'll test it more thoroughly, after that it can be merged. But feel free to review in the meantime.

File: discord/src/DiscordBot.jl [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    catch ex
+        @warn "Sending message failed" exception=(ex, catch_backtrace())
+    end
+end
+
+function main()
+    client = discord_client()
+    channels = load_channels()
+    for (start, talks) in load_talks()
+        wait = max(start - now(UTC), zero(Millisecond))
```
Comment (travigd) [inline discord/src/DiscordBot.jl:69]: Does this deal with talks in the past? Presumably we'd want to restart the bot if it dies in the middle of the conference.

Comment (christopher-dG) [reply discord/src/DiscordBot.jl]: Good catch!

**Comment by travigd** [comment]
LGTM.

File: data/discord.json [hunk 2]
```diff
@@ -3,23 +3,27 @@
     {
       "name": "General",
       "channelId": "709438882504245361",
+      "color": 4219864,
```
Comment (travigd) [inline data/discord.json:6]: @christopher-dG What is this value?

Comment (christopher-dG) [reply data/discord.json:6]: They're decimal color codes, this one is Julia Blue.

Comment (travigd) [reply data/discord.json:6]: Awesome. I will definitely use this elsewhere as well. Thanks!

File: discord/src/DiscordBot.jl [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    return sort(collect(buckets); by=first)
+end
+
+function format_message(talk, color)
+    description = talk.description
+    if length(description) > 2000
+        description = description[1:1994] * " [...]"
+    end
+
+    fields = [(; name="More info", value="https://juliacon2020.now.sh/live")]
```
Comment (christopher-dG) [inline discord/src/DiscordBot.jl:34]: @travigd Should this URL actually be https://live.juliacon.org/live ? I just realized that we were using that URL.

Comment (travigd) [reply discord/src/DiscordBot.jl:34]: Yep, it should. It'll autoredirect but yeah.

**Pull request merged**